### PR TITLE
check jobid consistency between input files in OAR

### PR DESCRIPTION
**Pull request by sfranky**
fix for bug https://github.com/qtop/qtop/issues/249

### CODE REVIEW

File: qtop.py [hunk 1]
```diff
@@ -2163,7 +2163,7 @@ class InvalidScheduler(Exception):
 
                 job_ids, user_names, job_states, job_queues = scheduling_system.get_jobs_info()
                 total_running_jobs, total_queued_jobs, qstatq_lod = scheduling_system.get_queues_info() # callthis elsewhere
-                worker_nodes = scheduling_system.get_worker_nodes()
+                worker_nodes = scheduling_system.get_worker_nodes(job_ids)
```
Comment (fgeorgatos) [inline qtop.py]: any impact on performance by these changes? (just asking...)

**Comment by sfranky** [comment]
not too much, I added a check in a list comprehension. if we have to state
the discrepancy explicitly though, it might impact the performance a little
bit.

**Comment by sfranky** [comment]
This PR should also result in some minor performance improvement, mostly memory-wise.

**Comment by fgeorgatos** [comment]
lgtm, tests pass and seems to work without issues (tbv)

**Pull request merged**